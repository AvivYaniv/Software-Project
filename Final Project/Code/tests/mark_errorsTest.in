mark_errors 0
solve tests/bgbgbg.txt
set 6 8 8
set 5 7 8
mark_errors 0
print_board
mark_errors 1
print_board
mark_errors 0
edit tests/bgbgbg.txt
set 6 8 8
set 5 7 8
mark_errors 0
print_board
mark_errors 1
solve tests/b2_2.txt #parameters check
print_board
mark_errors
mark_errors 0
mark_errors 		1 	 
mark_errors 	  0  
mark_errors aaaaa
mark_errors 0d
mark_errors b
exit