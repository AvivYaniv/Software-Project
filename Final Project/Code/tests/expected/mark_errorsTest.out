Sudoku
------
Enter your command:
ERROR: invalid command
Enter your command:
----------------------------------------
|            |          7.|      9.  8.|
|  5.  4.  6.|  8.      9.|      3.    |
|          8.|          3.|      4.    |
----------------------------------------
|            |          5.|      7.  9.|
|  6.        |          8.|            |
|      8.    |          1.|            |
----------------------------------------
|  3.  6.    |          4.|            |
|  4.  5.    |  7.        |      8.  1.|
|  8.        |      9.  2.|            |
----------------------------------------
Enter your command:
----------------------------------------
|            |          7.|      9.  8.|
|  5.  4.  6.|  8.      9.|      3.    |
|          8.|          3.|      4.    |
----------------------------------------
|            |          5.|      7.  9.|
|  6.        |          8.|            |
|      8.    |          1.|            |
----------------------------------------
|  3.  6.    |          4.|            |
|  4.  5.    |  7.      8*|      8.  1.|
|  8.        |      9.  2.|            |
----------------------------------------
Enter your command:
----------------------------------------
|            |          7.|      9.  8.|
|  5.  4.  6.|  8.      9.|      3.    |
|          8.|          3.|      4.    |
----------------------------------------
|            |          5.|      7.  9.|
|  6.        |          8.|            |
|      8.    |          1.|            |
----------------------------------------
|  3.  6.    |      8*  4.|            |
|  4.  5.    |  7.      8*|      8.  1.|
|  8.        |      9.  2.|            |
----------------------------------------
Enter your command:
Enter your command:
----------------------------------------
|            |          7.|      9.  8.|
|  5.  4.  6.|  8.      9.|      3.    |
|          8.|          3.|      4.    |
----------------------------------------
|            |          5.|      7.  9.|
|  6.        |          8.|            |
|      8.    |          1.|            |
----------------------------------------
|  3.  6.    |      8   4.|            |
|  4.  5.    |  7.      8 |      8.  1.|
|  8.        |      9.  2.|            |
----------------------------------------
Enter your command:
Enter your command:
----------------------------------------
|            |          7.|      9.  8.|
|  5.  4.  6.|  8.      9.|      3.    |
|          8.|          3.|      4.    |
----------------------------------------
|            |          5.|      7.  9.|
|  6.        |          8.|            |
|      8.    |          1.|            |
----------------------------------------
|  3.  6.    |      8*  4.|            |
|  4.  5.    |  7.      8*|      8.  1.|
|  8.        |      9.  2.|            |
----------------------------------------
Enter your command:
Enter your command:
----------------------------------------
|            |          7 |      9   8 |
|  5   4   6 |  8       9 |      3     |
|          8 |          3 |      4     |
----------------------------------------
|            |          5 |      7   9 |
|  6         |          8 |            |
|      8     |          1 |            |
----------------------------------------
|  3   6     |          4 |            |
|  4   5     |  7         |      8   1 |
|  8         |      9   2 |            |
----------------------------------------
Enter your command:
----------------------------------------
|            |          7 |      9   8 |
|  5   4   6 |  8       9 |      3     |
|          8 |          3 |      4     |
----------------------------------------
|            |          5 |      7   9 |
|  6         |          8*|            |
|      8     |          1 |            |
----------------------------------------
|  3   6     |          4 |            |
|  4   5     |  7       8*|      8*  1 |
|  8         |      9   2 |            |
----------------------------------------
Enter your command:
----------------------------------------
|            |          7 |      9   8 |
|  5   4   6 |  8       9 |      3     |
|          8 |          3 |      4     |
----------------------------------------
|            |          5 |      7   9 |
|  6         |          8*|            |
|      8     |          1 |            |
----------------------------------------
|  3   6     |      8*  4 |            |
|  4   5     |  7       8*|      8*  1 |
|  8         |      9   2 |            |
----------------------------------------
Enter your command:
ERROR: invalid command
Enter your command:
----------------------------------------
|            |          7 |      9   8 |
|  5   4   6 |  8       9 |      3     |
|          8 |          3 |      4     |
----------------------------------------
|            |          5 |      7   9 |
|  6         |          8*|            |
|      8     |          1 |            |
----------------------------------------
|  3   6     |      8*  4 |            |
|  4   5     |  7       8*|      8*  1 |
|  8         |      9   2 |            |
----------------------------------------
Enter your command:
ERROR: invalid command
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
ERROR: invalid command
Enter your command:
Enter your command:
Enter your command:
Enter your command:
Error: the value should be 0 or 1
Enter your command:
Error: the value should be 0 or 1
Enter your command:
Error: the value should be 0 or 1
Enter your command:
Exiting...
